integration-tests/src/test/kotlin/org/stellar/anchor/platform/Sep24Tests.kt

```diff
@@ -0,0 +1,48 @@
+@file:Suppress("UNCHECKED_CAST")
+
+package org.stellar.anchor.platform
+
+import org.junit.jupiter.api.Assertions.*
+import org.stellar.anchor.util.Sep1Helper
+
+lateinit var sep24Client: Sep24Client
+
+const val withdrawJson =
+  """{
+    "asset_code": "USDC",
+    "account": "GAIUIZPHLIHQEMNJGSZKCEUWHAZVGUZDBDMO2JXNAJZZZVNSVHQCEWJ4",
+    "lang": "en"
+}"""
+
+fun sep24TestAll(toml: Sep1Helper.TomlContent, jwt: String) {
+  println("Performing SEP24 tests...")
+  sep24Client = Sep24Client(toml.getString("TRANSFER_SERVER_SEP0024"), jwt)
+
+  testSep24TestInfo()
+  testSep24PostInteractive()
+}
+
+fun testSep24TestInfo() {
+  printRequest("Calling GET /info")
+  val info = sep24Client.getInfo()
+  printResponse(info)
+  assertTrue(info.deposit.isNotEmpty())
+  assertTrue(info.withdraw.isNotEmpty())
+}
+
+fun testSep24PostInteractive() {
+  printRequest("POST /transactions/withdraw/interactive")
+  val withdrawRequest = gson.fromJson(withdrawJson, HashMap::class.java)
+  val txn = sep24Client.withdraw(withdrawRequest as HashMap<String, String>)
```

Comment:
Thanks for the suggestion. I am half-way done because I need to switch to the MGI branch (core-release-1.0). 

I plan to add more tests to use the `withdrawJson` variable. 😁